cmd/kots/cli/install.go

```diff
@@ -330,8 +331,20 @@ func InstallCmd() *cobra.Command {
 			}()
 
 			// Possibly Successful Deployment - Report here
-			CollectFinishInstallMetrics(im)
 			if v.GetBool("port-forward") && !deployOptions.ExcludeAdminConsole {
+				ret := true
+				f := func() bool {
```

Comment:
This function will keep reporting success forever once the app is running.  Success needs to be reported once.  After timeout, failure needs to be reported once.  After that nothing else should be reported again.  If you change this to a loop in a go-routine, like @salahalsaleh is suggesting, this logic will be easy to write.

This should be a loop in a go-routine so it can be controlled better.  Also after timeout if app is not ready